src/qt/locale/bitcoin_fr.ts

```diff
@@ -4421,75 +4421,75 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Bitcoin Core and Dogecoin Core</source>
-        <translation type="unfinished"></translation>
+        <translation>Bitcoin Core and Dogecoin Core</translation>
```

Comment:
I would have expected at least `and` -> `et` here, but probably not translating `Core` to something else.